body.tex

```diff
@@ -32,47 +36,44 @@ \section{Network security}\label{sec:net}
+ \rec{IT}{netsec}{IT shall purchase routers capable of performing AES\cite{aes} IPSec in between Chile and SLAC.}
+{At lease two routers will be needed and we may need four for failover. This is a multi million dollar commitment. }
 
-Currently alert images from the base are to be transmitted using BBCP\footnote{BBCP is an alternative to Gridftp when transferring large amounts of data}. The control channel is encrypted but the data channel is open - this is considered secure in the scientific world \citep{bbcp}. Hence theoretically packets snooped on their way to SLAC  could be extracted and processed.
+If we do not transfer embargoed images to France or UK we understand encryption on the international links should not be needed.
 
 
-Our preference would be to do this in software using TLS based protocols such as HTTPS or FTPS to replace BBCP. This may cause some performance hit.
-We could encrypt the data  channel. This would cost us in  compute but could be similar to compression and not push up the number of cores needed for transfer. We would also have some software modification and setup there are projects like WireGuard\footnote{\url{https://www.wireguard.com/}} intending to do this at the kernel level but they are not suitable for Rubin use yet.
-There are also  examples of hardware solutions for this which would probably be affordable, INRIA in \cite{10.1007/978-3-642-45073-0_1} have build a FPGA based AES\cite{aes} implementation  which gives 100Gbit/s network rates with $\mu s$ additional latency. Table 3 of \cite{10.1007/978-3-642-45073-0_1} lists the hardware which looks modest enough - we work with INRIA already. There is no price given in the document, but considering the cost of components and a contract to put it together one \emph{might} consider it to be  under \$1M, but we would need to get a quote. This is probably more complex than we might want to get into.
 
-We could encrypt the images themselves before transmission or even on the FPGA of the camera data acquisition (DAQ) system.  The DAQ is complex construction prone to delays so we would rather not jeopardize the project by introducing changes there. Encrypting the files outside would mean more CPU and I/O - it would add a delay in the alerts processing - it may take as much as 20\% of the alerts time budget (1 minute).  This would not be very welcome in the science community. This would require some effort on our side to add encryption to the processing chain but this should be order some hundreds \$K depending on how \emph{secure} it is required to be.
+\section {Physical Security}
 
-Should we need to keep BBCP we would have to turn to IPsec on the router and this would require router upgrades. See also \citeds{DMTN-163} for more encryption background.
+On the mountain and in the base facility LSST networks and computers are in rooms requiring ID card access and the compounds have 24/7 security staff. We can increase physical security in Chile and SLAC.
```

Comment:
should be "...base facility, Rubin networks ..."

Reply:
yes Rubin better